sktime/base/_tests_and_checks.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""collection of different verificiation and checking classes that can be used in composition."""
```

Comment:
Thank you for adding _tests_and_checks.py Which assumption we have on the input data and thus which checks we need.